boards/openmote-b/Makefile.include

```diff
@@ -9,27 +9,32 @@ PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbserial*)))
 # Set default flash tool
 export PROGRAMMER ?= cc2538-bsl
 
+FLASHFILE ?= $(HEXFILE)
+RESET_FLAGS ?= $(BINDIR)
+export OBJDUMPFLAGS += --disassemble --source --disassembler-options=force-thumb
+
 ifeq ($(PROGRAMMER),cc2538-bsl)
   OS := $(shell uname)
   ifeq ($(OS),Linux)
     PORT_BSL ?= $(PORT_LINUX)
   else ifeq ($(OS),Darwin)
     PORT_BSL ?= $(PORT_DARWIN)
   endif
-  FLASHER = $(RIOTBASE)/dist/tools/cc2538-bsl/cc2538-bsl.py
-  FFLAGS  = -p "$(PORT_BSL)" --bootloader-invert-lines -e -w -v -b 460800 $(FLASHFILE)
+  FLASHER ?= $(RIOTBASE)/dist/tools/cc2538-bsl/cc2538-bsl.py
+  FFLAGS  ?= -p "$(PORT_BSL)" --bootloader-invert-lines -e -w -v -b 460800 $(FLASHFILE)
+  RESET ?= $(RIOTTOOLS)/cc2538-bsl/cc2538-bsl.py -p "$(PORT_BSL)" --bootloader-invert-lines
 else ifeq ($(PROGRAMMER),jlink)
-  FLASHER = $(RIOTBOARD)/$(BOARD)/dist/flash.sh
```

Comment:
Wait this was never there ?

Reply:
Yup. Also the problem is in the openmote-cc2538.  When I did the port I just copied...

Reply:
…